examples/step-59/step-59.cc

```diff
@@ -997,8 +998,10 @@ namespace Step59
                  << std::endl;
     time.restart();
 
-    ConstraintMatrix dummy;
-    dummy.close();
+    AffineConstraints<double> dummy_double;
+    dummy_double.close();
+    AffineConstraints<float> dummy_float;
+    dummy_float.close();
```

Comment:
... This gets a bit messy in step-59 where we now need two dummy `AffineConstraints` objects.